articles/java/spring-framework/configure-spring-cloud-stream-binder-java-app-azure-event-hub.md

```diff
@@ -77,6 +77,7 @@ Use the following steps to configure your application to produce and consume mes
     spring.cloud.azure.eventhubs.namespace=${AZURE_EVENTHUBS_NAMESPACE}
     spring.cloud.azure.eventhubs.processor.checkpoint-store.account-name=${AZURE_STORAGE_ACCOUNT_NAME}
     spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name=${AZURE_STORAGE_CONTAINER_NAME}
+    spring.cloud.azure.eventhubs.processor.checkpoint-store.account-key=${AZURE_STORAGE_ACCCOUNT_KEY}
```

Comment:
Hi @sayapeg thanks for creating the PR. While for this quickstart, we are leveraging passwordless authentication so we don't expose the account key property for Storage Blob, just like the connection string property for Event Hubs. 
May I know whether you have any problems running this quickstart without setting the account key property?

Reply:
And @chenrujun , we should also remove the line 96 about introducing the account key property, that might be confusing here.

Reply:
@yiliuTo Thank you for your confirm.

@sayapeg After you tried passwordless connection, could you please help to remove all `account-key` instead of adding it?

Reply:
@chenrujun @yiliuTo Initially, I initially got this error `"Unauthorized access. 'Listen' claim(s) are required to perform this operation.` for event hubs and `Status code 403, "﻿<?xml version="1.0" encoding="utf-8"?><Error><Code>AuthorizationPermissionMismatch</Code><Message>This request is not authorized to perform this operation using this permission.
RequestId:179b01e5-f01e-0072-4c78-6d2377000000
Time:2023-04-12T19:55:52.2652351Z</Message></Error>"` for storage blob. During that process, I saw that event hub auto-configuration considers the presence of the property `connection-string`. After looking at the table describing the list of properties that should be in the configuration, I added `connection-string` and, similarly, `account-key` for the checkpoint store, thinking that the missing property caused the above error. But I didn't know then that even without the `connection-string`, the application got the token using `AzureCliCredential` as I did an `az login` for something else. And for the configured event hub and the storage account, I have Contributor access.